opentreemap/manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3.7
```

Comment:
This change is connected to the changes and justification included in: https://github.com/OpenTreeMap/otm-cloud/pull/501/files#r363948469.

Reply:
Thanks. I agree with the changes you made in OpenTreeMap/otm-cloud/pull/501